floor/floor/controller/controller.py

```diff
@@ -81,8 +78,8 @@ def square_weight_off(self, index):
         self.SYNTHETIC_WEIGHTS[index] = 0
 
         # Scan the weighs and see if anything is still set
-        for index in self.SYNTHETIC_WEIGHTS:
-            if self.SYNTHETIC_WEIGHTS[index]:
+        for value in self.SYNTHETIC_WEIGHTS:
+            if value:
```

Comment:
Dev server changed MAX_FLOOR_VALUE from 1 to 1023, which uncovered this boneheaded bug of mine.

Reply:
whoops! thanks!